### PR TITLE
Incorporate README Badges (#178)

### LINKED ISSUE #178: Incorporate README Badges
In a branch that's now-defunct, I had made changes to add some nifty badges to our README. Since that branch is not going to be merged, those changes should be made on their own.

https://github.com/quintilesims/layer0/compare/documentation-guides
- Changes in README.md
- Changes in RELEASE.md

### PR DESCRIPTION
**Pull request by tlake**
Closes #178

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # ![logo](https://quintilesims.github.io/layer0/static/logo_rs.png)
 
+[![Build Status](https://travis-ci.org/quintilesims/layer0.svg?branch=master)](https://travis-ci.org/quintilesims/layer0)  [![Version](https://img.shields.io/badge/version-0.9.0-brightgreen.svg)](https://github.com/quintilesims/layer0/releases/tag/v0.9.0)  [![Documentation](https://img.shields.io/badge/docs-latest-blue.svg)](https://quintilesims.github.io/layer0/)
```
Comment (diemonster) [inline README.md]: Is there any way we could avoid having to hardcore the version here?

Comment (tlake) [inline README.md]: Huh, I don't know - it didn't occur to me to look into it. Do we have a pattern for templating `.md`s anywhere else in Layer0? I know we do some stuff with `L0_VERSION?=$(shell git describe --tags)` in our Makefiles.

Comment (diemonster) [inline README.md]: as per our discussion: let's add a new tag with every release called `latest` which we can use here

Comment (tlake) [inline README.md]: Will do - I'll pull in #183

Comment (tlake) [inline README.md]: Further reflection has prompted me to postpone the creation of a `latest` tag. Pushing two tags for the same release would likely wreak havoc wherever our scripts call `git describe --tags`. There's a Stack Overflow question [here](http://stackoverflow.com/questions/8089002/git-describe-with-two-tags-on-the-same-commit) that seems to imply that `git describe --tags` will take the newest tag, which means that in theory we could be sure to create the version-specific tag **after** the `latest` tag, but that seems fragile and janky.

So for now, we'll keep our cool badges and require a little bit of manual attention during the release process (it's documented in RELEASE.md) until we figure something better out.

- Removing #183 from this PR
- Referencing #117 for the future

Comment (diemonster) [inline README.md]: 👍

**Pull request merged**